packages/app/src/cli/utilities/developer-platform-client/app-management-client/graphql/create-app.ts

```diff
@@ -32,6 +33,7 @@ export interface CreateAppMutationSchema {
   appCreate: {
     app: {
       id: string
+      key: string
```

Comment:
do we need the `activeRoot` here to get the secret?

Reply:
Hmmmm, I guess for `dev` that might be relevant. It wouldn't be relevant in any other case I can think of, so it didn't come up in testing.

In the interest of shipping, I'll leave it out for now, and you should be able to add it in and test it in the context of `dev`.